### PR TITLE
Fixes for openSUSE:*ARM rebuilding

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by jberry-suse** [comment]
Travis failure tracked in openSUSE/open-build-service#4462

File: pkglistgen.py [hunk 1]
```diff
@@ -1120,6 +1121,9 @@ def update_and_solve_target(self, apiurl, target_project, target_config, main_re
         product_dir = os.path.join(cache_dir, product)
         release_dir = os.path.join(cache_dir, release)
 
+        conf_no_colon = conf.config['checkout_no_colon']
+        conf.config['checkout_no_colon'] = False
```
Comment (jberry-suse) [inline pkglistgen.py]: Is this not the [default](https://github.com/openSUSE/osc/blob/5e3fe8ba08d7625140f06441c804e82790ea7db0/osc/conf.py#L149)? Are you explicitly setting to override local config or what?

File: pkglistgen.py [hunk 2]
```diff
@@ -1120,6 +1121,9 @@ def update_and_solve_target(self, apiurl, target_project, target_config, main_re
         product_dir = os.path.join(cache_dir, product)
         release_dir = os.path.join(cache_dir, release)
 
+        conf_no_colon = conf.config['checkout_no_colon']
```
Comment (jberry-suse) [inline pkglistgen.py]: `conf_no_colon` never used.

File: pkglistgen.py (hunk 1 above)
Comment (jberry-suse) [reply pkglistgen.py]: Considering in a commit title "Skip automatic rebuild for more pattern packages" assuming this was not intentional?

Comment (dirkmueller) [reply pkglistgen.py]: sorry, this was unrelated. removed.

**Pull request merged**